test/python/circuit/test_scheduled_circuit.py

```diff
@@ -109,13 +109,6 @@ def test_cannot_schedule_circuit_with_mixed_SI_and_dt_when_no_one_tells_dt(self)
         with self.assertRaises(QiskitError):
             transpile(qc, self.backend_without_dt, scheduling_method='alap')
 
-    def test_can_execute_delay_circuit_when_schedule_circuit_off(self):
-        qc = QuantumCircuit(2)
-        qc.h(0)
-        qc.delay(500, 1)
-        qc.cx(0, 1)
-        execute(qc, backend=self.backend_with_dt, schedule_circuit=False)
```

Comment:
I don't know how this wasn't failing before, but I also don't know why it hangs rather than fails. (It hangs on `job = backend.run(qobj, **run_config)` of `execute.py`)

Reply:
Yeah, this test must be removed. I confirmed it hangs in my local env. I don't know why it doesn't fail or how it wasn't failing before, either.

Reply:
Thank you for checking it out!